code/modules/clothing/head/helmet.dm

```diff
@@ -74,6 +74,15 @@
 /obj/item/clothing/head/helmet/sec
 	can_flashlight = TRUE
 
+/obj/item/clothing/head/helmet/syndicate
+	can_flashlight = TRUE
+	name = "syndicate helmet"
+	desc = "Standar issue security helmet. Painted red to help with identification,additionally fitted with flash protection and muffling"
```

Comment:
```suggestion
	desc = "Standard issue security helmet. Painted red to help with identification, additionally fitted with flash protection and muffling"
```

Reply:
Additionally needs a full stop at the end of the sentance

Reply:
Resolved as well. Thank you for kindly reviewing my code